### DISCUSSION TITLE
Use sylius as a bundle

### DISCUSSION THREAD

**Discussion by ziat1988** [Sylius/Sylius#14907] [General]
I have a Symfony project developed since 2020 and I'm searching for a solution ecommerce. In the official doc of Sylius I only see the way to create a new project.
Is there a way to use Sylius to integrate into my exist project Symfony?
Thanks

**Comment by mbabker** [comment]
All of the Sylius components and bundles are installable standalone thanks to the git subsplit (basically the same way Symfony has its monorepo and all the packages can be installed separately).  The documentation is geared for a full-stack Sylius install, but with a little effort, you can pull in the bits and pieces you'd want to use in your application.
